source/includes/_configuration-options.md

```diff
@@ -141,30 +147,63 @@ If there is a need to use custom css stylesheet inside the Tipser dialog iframe,
 
 ***
 
+
+### Add To Cart Settings
+
+When the`directToCheckoutMode` option is set to `true`, after clicking the `buy button` in Product component you go directly to checkout, without displaying the product dialog.
+
```

Comment:
może warto by też napisać o zachowaniu directToCheckoutMode z poziomu dialogu? Kiedy jesteś w dialogu i klikniesz add-to-cart to to też przechodzisz od razu do dialog-checkoutu